inductiva/molecules/scenarios/md_water_box/post_processing.py

```diff
@@ -21,13 +22,14 @@ def __init__(self, sim_output_path: Path = None):
         self.sim_output_dir = sim_output_path
 
     def render(self):
-        """Visualize the simulation outputs in a notebook using NGLView."""
+        """Visualize the simulation outputs in a notebook widget 
+        using NGLView."""
```

Comment:
One-liner. The info about this being generated with NGLView isn't relevant to the user, so it can be moved to a second sentence/paragraph.